app/src/main/java/org/simple/clinic/home/overdue/OverdueScreenController.kt

```diff
@@ -72,7 +72,7 @@ class OverdueScreenController @Inject constructor(
                 phoneNumber = it.phoneNumber?.number,
                 bpSystolic = it.bloodPressure.systolic,
                 bpDiastolic = it.bloodPressure.diastolic,
-                bpDaysAgo = calculateDaysAgoFromInstant(it.bloodPressure.updatedAt),
+                bpDaysAgo = calculateDaysAgoFromInstant(it.bloodPressure.recordedAt),
```

Comment:
Let's add one associated controller test for this change?

Reply:
`OverdueListItem` is a UI related implementation detail which there is no point writing a unit test for. 
You can actually run a UI test on this instead.

Reply:
Yes we could add a UI test. Can we also not add a unit test that ensures that the controller calculates the `calculateDaysAgoFromInstant` using the `recordedAt` value instead of `updatedAt`? By having a recordedAt value that pre-dates updatedAt for instance.

Reply:
This is still a presentation detail.
I don't find any value adding an unnecessary test.

Reply:
Let me know if this makes sense. We can get on a call if you want to discuss this further.

Reply:
Can you please link me to the test that verifies that "`recordedAt` is used to fetch overdue patients and not `updatedAt`"?

Reply:
This is part of `AppointmentRepositoryAndroidTest`